componentLibraries/defaultLibrary/Signal/Control/SignalPID2.hpp

```diff
@@ -130,7 +130,7 @@ class SignalPID2 : public ComponentSignal
         mI = mI + mTimestep/mTt * (u-v);
 
         // Write output and remember error
-        (*mpOut) = v;
+        (*mpOut) = u;
```

Comment:
Since the parameters was named Umin and Umax I assumed they refered not only to the I-part but also to the output signal u. I would guess it's at least a common (most common?) use case to limit the output also. Third alternative might be to have separate limits for integrator and output. Better documentation in any way would sure help.

Reply:
The constants Umin and Umax should at least have a description. We could also make it an option whether or not the output signal should be limited or only the I-part.